File: todo/src/main/java/com/mammatustech/todo/HttpClient.java
package com.mammatustech.todo;

import io.advantageous.boon.json.JsonFactory;
import io.advantageous.qbit.http.HTTP;

public class HttpClient {

    public static void main(final String... args) throws Exception {

        for (int index = 0; index < 100; index++) {

            HTTP.postJSON("http://localhost:8888/v1/todo-service/todo",
                    JsonFactory.toJson(new Todo("name" + index, "desc" + index, System.currentTimeMillis() )));
            System.out.print(".");
        }
    }

}

File: todo-scala/src/main/scala/com/mammatustech/todo/HttpClient.scala
package com.mammatustech.todo

import io.advantageous.boon.json.JsonFactory
import io.advantageous.qbit.http.HTTP

object HttpClientExample {
  @throws(classOf[Exception])
  def main(args: String*) {

      var index: Int = 0
      while (index < 100) {
        {
          HTTP.postJSON("http://localhost:8888/v1/todo-service/todo",
            JsonFactory.toJson(
              new Todo("name" + index, "desc" + index,
                System.currentTimeMillis)))
            print(".")
        }
        {
          index += 1
        }
      }
    }

}